src/client/views/main.html

```diff
@@ -7,8 +7,14 @@
   <button id='logout' ng-click='logout()'>Logout</button>
   <div ui-view='restaurantList'>
   </div>
-  <div ui-view='map'>
+  <!-- This is now hard coding the map element rather than using a ui-view -->
+  <div>
+    <h4>MAP</h4>
+    <div id='map'>
+    </div>
   </div>
+  <!-- <div ui-view='map'>
```

Comment:
We're planning on refactoring back to this, so I'm going to leave it. This is just a temporary solution because of issues with being able to access $scope.